data.py

```diff
@@ -0,0 +1,55 @@
+from triumph import db
+from triumph.models import Category, Product, Feature, Sku, ImageUrl
+import json
+
+
+def get_or_create(model, **field):
+    instance = db.session.query(model).filter_by(**field).first()
+    if instance:
+        return instance
+    else:
+        instance = model(**field)
+        return instance
+
+
+db.create_all()
+
+with open('data.json') as data_file:
+    data = json.load(data_file)
+
+for raw_product in data:
+
+    category = get_or_create(Category, name=raw_product['category'][-2])
+    db.session.add(category)
+
+    feature = get_or_create(Feature, name=raw_product['feature'])
+    db.session.add(feature)
```

Comment:
do  'db.session.add()' in function 'get_or_create()' otherwise you will have to add object in session every time  after calling 'get_or_create()'.